src/main/java/net/glowstone/block/entity/state/GlowContainer.java

```diff
@@ -29,22 +37,23 @@ public GlowContainer(GlowBlock block) {
 
     @Override
     public String getLootTableName() {
-        return null;
+        return lootTable.get();
     }
 
     @Override
     public boolean hasLootTable() {
-        return false;
+        return lootTable.get() != null;
     }
 
     @Override
     public String setLootTable(String name) {
-        return null;
+        return setLootTable(name, 0);
     }
 
     @Override
     public String setLootTable(String s, long l) {
```

Comment:
Can you give the variables proper names here?

Reply:
Done in f2d3ca7